docs/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "scripts": {
+    "docs:dev": "vitepress dev",
+    "docs:build": "vitepress build",
+    "docs:preview": "vitepress preview"
+  },
+  "devDependencies": {
+    "vitepress": "^1.3.1",
+    "mermaid": "^10.9.0",
+    "vitepress-plugin-mermaid": "^2.0.16"
```

Comment:
I would encourage the use of pinned dependencies here (as renovatebot will pin them later anyways)